indico/modules/rb_new/controllers/backend.py

```diff
@@ -34,7 +35,9 @@
     'start_dt': fields.DateTime(),
```

Comment:
I know it's not part of this PR, but maybe calling this dict `room_args` or `room_request_args`? `request_args` sounds rather generic.

Reply:
Or just move it inside:

```python

class RHRoomBookingSearch(RHRoomBookingBase):
    @use_args({
        'capacity': fields.Int(),
        'room_name': fields.Str(),
        'start_dt': fields.DateTime(),
        'end_dt': fields.DateTime(),
        'repeat_frequency': EnumField(RepeatFrequency),
        'repeat_interval': fields.Int(missing=0),
        'building': fields.Str(),
        'floor': fields.Str()
    })
    def _process(self, args):
        rooms = sorted(search_for_rooms(args), key=lambda r: natural_sort_key(r.full_name))
        return rooms_schema.dumps(rooms).data
```